src/utils.js

```diff
@@ -27,19 +29,29 @@ const setupCml = async opts => {
     }
   }
 
-  await exec(`${sudoPath} npm config set user 0`);
+  try {
+    const cmlVer = await exec('echo none | cml --version');
```

Comment:
Arcane code golf alternatives 😄 
```suggestion
    const cmlVer = await exec(':|cml --version');
```
```suggestion
    const cmlVer = await exec('cml --version<&-');
```

Reply:
might include a [link](https://www.gnu.org/software/bash/manual/bash.html#Redirections) for the others who may try their hand at contributing and arent bash ninjas